Result of `nixpkgs-review pr 107990` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>python37Packages.pyipp</li>
    <li>python37Packages.pysvn</li>
    <li>python38Packages.pyipp</li>
    <li>python38Packages.pysvn</li>
    <li>python39Packages.pyipp</li>
    <li>python39Packages.pysvn</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python37Packages.deepmerge</li>
    <li>python37Packages.vcver</li>
    <li>python38Packages.deepmerge</li>
    <li>python38Packages.vcver</li>
    <li>python39Packages.deepmerge</li>
    <li>python39Packages.vcver</li>
  </ul>
</details>

pysvn
```
  Compile: generate_svn_error_codes/generate_svn_error_codes.cpp into generate_svn_error_codes.o                                                                                              /nix/store/rdbq2q75ysh13gis328k7vwwf4s22hvw-bash-4.4-p23/bin/bash: g++: command not found                                                                                                   make: *** [Makefile:203: generate_svn_error_codes.o] Error 127
```

pyipp fails because of sandbox.

--- a/pkgs/development/python-modules/pysvn/default.nix
+++ b/pkgs/development/python-modules/pysvn/default.nix
@@ -60,10 +60,10 @@ buildPythonPackage rec {
     sed -i "s|/bin/bash|${bash}/bin/bash|" ../Tests/test-*.sh
     make -C ../Tests
 
+    ${python.interpreter} -c "import pysvn"
+
     runHook postCheck
   '';
-
-  pythonImportCheck = [ "pysvn" ];
 
   installPhase = ''
     dest=$(toPythonPath $out)/pysvn
@@ -80,5 +80,4 @@ buildPythonPackage rec {
     homepage = "http://pysvn.tigris.org/";
     license = licenses.asl20;
   };
-
 }

--- a/pkgs/development/python-modules/vcver/default.nix
+++ b/pkgs/development/python-modules/vcver/default.nix
@@ -20,7 +20,7 @@ buildPythonPackage rec {
   # circular dependency on test tool uranium https://pypi.org/project/uranium/
   doCheck = false;
 
-  pythonImportTests = [ "vcver" ];
+  pythonImportsCheck = [ "vcver" ];
 
   meta = with lib; {
     description = "Reference Implementation of vcver";

--- a/pkgs/tools/security/gospider/default.nix
+++ b/pkgs/tools/security/gospider/default.nix
@@ -10,7 +10,7 @@ buildGoModule rec {
   src = fetchFromGitHub {
     owner = "jaeles-project";
     repo = pname;
-    rev = "${version}";
+    rev = version;
     sha256 = "03gl8y2047iwa6bhmayyds3li21wy3sw1x4hpp9zgqgi95039q86";
   };
 

